### PR TITLE
Run crossdock with all-in-one

### PR DESCRIPTION
**Pull request by bhavin192**
Reference: jaegertracing/jaeger#1762

### CODE REVIEW

**Comment by bhavin192** [comment]
One test is still failing, I will try to investigate this later today.

File: jaeger-crossdock/docker-compose.yml [hunk 1]
```diff
@@ -35,40 +35,73 @@ services:
...
     nodejs:
         image: jaegertracing/xdock-node
+        links:
+            - "jaeger:jaeger-agent"
         ports:
             - "8080-8082"
 
     java-udp:
         depends_on:
             - test_driver
```
Comment (yurishkuro) [inline jaeger-crossdock/docker-compose.yml]: I think this needs to depend on `jaeger`, if you look at PRs for the other clients

Comment (bhavin192) [reply jaeger-crossdock/docker-compose.yml]: Yes, you are right. I think earlier it was like `jave-udp -> test_driver -> (jaeger components)`.

File: jaeger-crossdock/docker-compose.yml [hunk 2]
```diff
@@ -35,40 +35,74 @@ services:
...
+            - jaeger
         ports:
             - "8080-8082"
         environment:
             - SENDER=udp
 
     java-http:
         build: .
+        links:
+            - "jaeger:jaeger-agent"
```
Comment (yurishkuro) [inline jaeger-crossdock/docker-compose.yml]: Did you get the test pass locally? Failure looks specific to the change.

Java http actor would be trying to submit spans to collector, so the renaming probably needs to be to jaeger-collector.

Comment (bhavin192) [reply jaeger-crossdock/docker-compose.yml]: Yep, you are right. With suggested change I got it working locally

**Pull request merged**